plugins/course-apps/live/BbbSettings.test.jsx

```diff
@@ -4,21 +4,16 @@ import {
   getByRole,
   getAllByRole,
   waitForElementToBeRemoved,
-} from '@testing-library/react';
+  initializeMocks,
+} from 'CourseAuthoring/testUtils';
 
 import ReactDOM from 'react-dom';
-import { Routes, Route, MemoryRouter } from 'react-router-dom';
-import { initializeMockApp } from '@edx/frontend-platform';
-import MockAdapter from 'axios-mock-adapter';
-import { getAuthenticatedHttpClient } from '@edx/frontend-platform/auth';
-import { AppProvider, PageWrap } from '@edx/frontend-platform/react';
-import { IntlProvider } from '@edx/frontend-platform/i18n';
 
 import userEvent from '@testing-library/user-event';
-import initializeStore from 'CourseAuthoring/store';
 import { executeThunk } from 'CourseAuthoring/utils';
 import PagesAndResourcesProvider from 'CourseAuthoring/pages-and-resources/PagesAndResourcesProvider';
 
+import { CourseAuthoringProvider } from '@src/CourseAuthoringContext';
```

Comment:
```suggestion
import { CourseAuthoringProvider } from 'CourseAuthoring/CourseAuthoringContext';
```
Since the `plugins` are supposed to be somewhat separate, we use `CourseAuthoring/` instead of `@src/` in that folder.

Reply:
Updated: https://github.com/openedx/frontend-app-authoring/pull/2724/commits/8f0fc17c7e4d2064b658d6dc9ac7cbca9febd875